code/modules/modular_computers/computers/item/pda.dm

```diff
@@ -185,6 +185,18 @@
 		update_appearance()
 		playsound(src, 'sound/machines/pda_button2.ogg', 50, TRUE)
 
+/obj/item/modular_computer/pda/proc/swap_pen(mob/user, obj/item/tool)
+
+	if(issilicon(user) || !user.can_perform_action(src, FORBID_TELEKINESIS_REACH)) //TK doesn't work even with this removed but here for readability
+		return
+
```

Comment:
```suggestion
```
It isn't even needed as your own comment points it out. Furthermore, if it were, it'd have to be a few operations before the item is transferred into the PDA, otherwise you'd end up with two pens inside it.

Reply:
This is not even my comment... I just did it based on remove_pen()